finish/readme.md

```diff
@@ -0,0 +1,48 @@
+# Setup
+
+To use the sample application, download and extract the [sampleapp_mpconfig.zip] 
+(https://github.com/OpenLiberty/iguide-microprofile-config/raw/master/finish/sampleapp_mpconfig.zip) 
+file to your local directory.
+
+Use the `mvn install` Maven command from the directory that contains the extracted .zip files 
+to build the project and install it in your local repository. The command creates the 
+`displayCarTypes/target/liberty` directory that contains your Liberty server, mpConfigServer, 
+and starts the server in the background.
+
+To stop the running server, run the Maven command `mvn liberty:stop-server`. To start
+the mpConfigServer, run the Maven command `mvn liberty:start-server`.
+
+To access the sample application, visit the http://localhost:9080/car-types URL. Initially, this will 
+show all 5 car types, SUV, Crossover, Coupe, Truck, and Convertible, since the bootstrap.properties
+contains the port value 9083. 
+
+The <extract-directory>/displayCarTypes/src directory contains the 
+InventoryConfig.java file as shown throughout this guide.  This file is where the 
+port configuration property value is injected into the code. The variable has the 
```

Comment:
I think it is better to say "This file is where the port configuration property and its value is injected into the code".